Resources/MessageQueue/StompMQConnector.py

```diff
@@ -20,24 +20,33 @@ class StompMQConnector(MQConnector):
   Class for management of message queue connections
   Allows to both send and receive messages from a queue
   """
-
-  def __init__(self, parameters={}):
+  # Setting for the reconnection handling by stomp interface.
+  # See e.g. the description of Transport class in
+  # https://github.com/jasonrbriggs/stomp.py/blob/master/stomp/transport.py
+  RECONNECT_SLEEP_INITIAL = 1  # [s]  Initial delay before reattempting to establish a connection.
+  RECONNECT_SLEEP_INCREASE = 0.5  # Factor by which sleep delay is increased 0.5 means increase by 50%.
+  RECONNECT_SLEEP_MAX = 120  # [s] The maximum delay that can be reached independent of increasing procedure.
+  RECONNECT_SLEEP_JITTER = 0.1  # Random factor to add. 0.1 means a random number from 0 to 10% of the current time.
+  RECONNECT_ATTEMPTS_MAX = 1e4  # Maximum attempts to reconnect.
+
+  PORT = 61613
+
+  def __init__(self, parameters=None):
     """ Standard constructor
     """
     super(StompMQConnector, self).__init__()
     self.log = gLogger.getSubLogger(self.__class__.__name__)
-    self.parameters = parameters.copy()
     self.connections = {}
 
     if 'DIRAC_DEBUG_STOMP' in os.environ:
       gLogger.enableLogsFromExternalLibs()
 
   def setupConnection(self, parameters=None):
-    #"""
-    # Establishes a new connection to a Stomp server, e.g. RabbitMQ
-    #:param dict parameters: dictionary with additional MQ parameters if any
-    #:return: S_OK/S_ERROR
-    #"""
+    """
+     Establishes a new connection to a Stomp server, e.g. RabbitMQ
+    :param dict parameters: dictionary with additional MQ parameters if any
```

Comment:
you need a space between the line and the param, and with the return

Reply:
I changed it to Args/Return version.